requirements.txt

```diff
@@ -1,2 +1,70 @@
+altgraph==0.10.2
```

Comment:
Your requirements.txt file has grown a lot more than it needs to.

It should only need to be:
```
Flask==1.1.1
requests==2.24.0
python-dotenv==0.12.0
```

Have you possibly not been running the project inside venv, so you've included all python packages you've installed globally on your machine?